openwisp_radius/tasks.py

```diff
@@ -231,3 +231,32 @@ def get_radius_attributes(user):
                 f' RadiusAccounting object of "{user}" user'
             )
     RadiusAccounting.objects.bulk_update(updated_sessions, fields=["groupname"])
+
+
+@shared_task(base=OpenwispCeleryTask)
+def handle_accounting_on(organization_id, nas_ip_address):
+    """
+    Handle Accounting-On packets by closing stale sessions for the NAS.
+
+    This task is executed asynchronously to avoid blocking the API response
+    and to prevent performance issues when there are many sessions to close.
+
+    Args:
+        organization_id: UUID of the organization
+        nas_ip_address: IP address of the NAS
+
+    Returns:
+        Number of sessions closed
+    """
```

Comment:
This task is pretty small, so this verbose docstring is definitely overkill. I'd move any useful info to the docstring of the model class method and remove the docstring entirely here.